Makefile

```diff
@@ -2,7 +2,7 @@ INSTALL	:= install
 DESTDIR	:= /
 PREFIX	:= /usr
 PACKAGE := openvpn-netfilter
-VERSION := 1.0.0
+VERSION := 1.0.1
```

Comment:
should probably bump this separately but ill just approve this time anyway, it deserves a bump at this point